### PR TITLE
[DDW-765]: Change behavior of system dialogs

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR updated behavior of system dialogs
[YouTrack Ticket](https://iohk.myjetbrains.com/youtrack/issue/DDW-765)

## Todos
- [x] Update behavior of system dialogs

## Screenshots
image
image
image
image

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1564415536076900)
- [x] Click on file dialog buttons on all screens to open file open/save dialogs and check if they are always on top of main window 

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by yakovkaravelov** [comment]
> @yakovkaravelov please, review these errors
> 
> image

@daniloprates Would you please give me the file you tried?

**Comment by yakovkaravelov** [comment]
@daniloprates 
Updated this.
Please review again.
Thanks.

**Comment by daniloprates** [comment]
@yakovkaravelov the drag & drop functionality is not available anymore:

### Before
bf

### After 
af

**Comment by daniloprates** [comment]
@MarcusHurney could you review this as well, as it touches sensitive parts of the code.

**Comment by yakovkaravelov** [comment]
@daniloprates @MarcusHurney 
Fixed all the issues Danilo posted.
Please review again.
Thanks.

**Comment by daniloprates** [comment]
On hold until Buildkite is fixed.

**Comment by daniloprates** [comment]
The solution for the broken E2E test is good for the current state of the App. The only other place we had to find a solution for the Electron dialog is in the Paper Wallet, where we completely ignore the Dialog and [manually set](https://github.com/input-output-hk/daedalus/blob/develop/features/tests/e2e/steps/paper-wallets-certificate-steps.js#L16) the value in the store, which is hackier. I suggest we approve this PR and find a better solution for these 2 cases and possible future ones, where we can mock the dialog and send the desirable value. cc @nikolaglumac

**Comment by nikolaglumac** [comment]
@DominikGuzei please see to review this one today 🙏 🙏 🙏

File: features/tests/e2e/setup/electron.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import path from 'path';
 import { Application } from 'spectron';
+import electron, { dialog } from 'electron';
```
[the remark this answers is not in the document]
Comment (nikolaglumac) [reply features/tests/e2e/setup/electron.js]: @DominikGuzei I have sorted it out here: feature/ddw-765-change-system-dialogs

**Comment by nikolaglumac** [comment]
> Great work @daniloprates 👍 the only negative observation is that the dropzone does not highlight itself on drag-over and you just see a "plus icon" on the whole app, although you can only drop the files on the button … im not sure how it looked before but that is improvable imo ;) if that's expected then this PR is ready for merge!

@DominikGuzei the import wallet dialog is not used in production and we don't have this drop zone enabled anywhere else in the app. Your comment is perfectly valid but there is very little value in fixing this small UX issue at the moment.

**Pull request merged**